### PR TITLE
added a warning for grafana removal

### PR DESCRIPTION
**Pull request by a-thaler**
**Description**

Changes proposed in this pull request:

- Introduced a new "Deprecated monitoring" navigation to "Observability" listing the deprecated prometheus instances
- Added a cluster details banner in case a "monitoring-auth-proxy-grafana-user" secret is available to inform that grafana will be gone
- ...

**Related issue(s)**

### CODE REVIEW

File: extensions/configuration/monitoring.yaml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+      group: monitoring.coreos.com
+      version: v1
+      namespace: kyma-system
+    category: Observability
+    scope: cluster
+    name: Deprecated Monitoring
+    filter: 'data.metadata.name="monitoring-prometheus"'
+    urlPath: prometheuses
+    description: >-
+      Disclaimer: Access to Kyma's Grafana/Prometheus is {{[discontinued](https://blogs.sap.com/2022/12/09/deprecation-of-prometheus-grafana-based-monitoring-in-sap-btp-kyma-runtime)}}!
```
Comment (NHingerl) [inline extensions/configuration/monitoring.yaml:36]:
```suggestion
      Remember: Default access to Kyma's Grafana/Prometheus is {{[discontinued](https://blogs.sap.com/2022/12/09/deprecation-of-prometheus-grafana-based-monitoring-in-sap-btp-kyma-runtime)}} - set up your desired monitoring solution now!
```

File: extensions/configuration/monitoring.yaml [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+      targets:
+        - slot: details-top
+          location: ClusterOverview
+      order: 1
+      children:
+        - widget: Alert
+          source: "'deprecation.long'"
+          severity: warning
+        - widget: ExternalLink
+          name: "Deprecation Blog Post"
```
Comment (NHingerl) [inline extensions/configuration/monitoring.yaml:54]:
```suggestion
          name: "Read the deprecation blog post"
```

File: extensions/configuration/monitoring.yaml [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    - name: Namespace
+      source: metadata.namespace
+    - name: Kind
+      source: kind
+    - source: '"deprecation.short"'
+      widget: Alert
+      severity: warning
+  translations: |-
+    en: 
+        deprecation.title: Grafana/Prometheus Removal on November 18, 2023
```
Comment (NHingerl) [inline extensions/configuration/monitoring.yaml:81]:
```suggestion
        deprecation.title: Prepare for the Grafana/Prometheus Removal on November 18, 2023
```

File: extensions/configuration/monitoring.yaml [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+        deprecation.long: |
+          You are actively exposing Kyma's Grafana instance using below Secret.
```
Comment (NHingerl) [inline extensions/configuration/monitoring.yaml:83]:
```suggestion
          You will be affected by the removal of Kyma's Grafana/Prometheus instance!
          Set up your desired monitoring solution now and delete the Secret you are currently using to expose Kyma's Grafana instance.
```

File: extensions/configuration/monitoring.yaml [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+          The access to Kyma's Grafana/Prometheus has been deprecated since January 20, 2023 and is scheduled to be removed on November 18, 2023.
+          Further details can be found in the blog post linked below.
```
Comment (NHingerl) [inline extensions/configuration/monitoring.yaml:85]:
```suggestion
          To learn more about the impact and what you can do, read the blog post.
```

File: extensions/configuration/monitoring.yaml [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+        deprecation.short: Access to Kyma's Grafana/Prometheus is discontinued, this instance will be removed on November 18, 2023!
```
Comment (NHingerl) [inline extensions/configuration/monitoring.yaml:86]:
```suggestion
        deprecation.short: Access to Kyma's Grafana/Prometheus is discontinued. This instance will be removed on November 18, 2023!
```

**Review by NHingerl — COMMENTED** [review]
not sure if the text fits into the available UI; let me know if it looks weird

**Review by NHingerl — APPROVED** [review]

**Review by mrCherry97 — APPROVED** [review]

**Pull request merged**